mcp/gcp_cloudrun.tf

```diff
@@ -0,0 +1,162 @@
+
+//noinspection HILUnresolvedReference
+data "google_project" "default" {
+  count      = lookup(local.cloudrun, "create_google_project", false) ? 0: 1
+  project_id = lookup(local.cloudrun, "project_id", "")
+}
+
+//noinspection HILUnresolvedReference
+data "google_organization" "default" {
```

Comment:
You don't need to go to this level. I mentioned project because in a nu bet of cases the project is just a wrapper around and application to isolate and control resources. Organizations are also not technically a requirement